cluster/node-pools/master-default/userdata.yaml

```diff
@@ -139,6 +139,11 @@ write_files:
           - --audit-log-maxage=0
           - --audit-log-maxbackup=0
           - --audit-policy-file=/etc/kubernetes/config/audit-policy.yaml
+          {{ else if and (eq .Cluster.ConfigItems.enable_default_sa "true") (eq .Cluster.ConfigItems.auditlog_read_access "true") }}
+          - --audit-log-path=/dev/stdout
+          - --audit-log-maxage=0
+          - --audit-log-maxbackup=0
+          - --audit-policy-file=/etc/kubernetes/config/audit-policy-ro.yaml
```

Comment:
Can this coexist with the other auditlog configurations?

Reply:
We have `audittrail_url` set in one of the clusters, for example, just to test the whole thing (adapter + resender).

Reply:
Ah right. I missed that. Let me change it so it supports both if possible.

Reply:
If that's too complicated, we could also make them mutually exclusive and just chase people in the test cluster to migrate off. We don't change the audittrail adapter very often anyway.

Reply:
I changed it so it can work together, only thing is we would now log all audit events to the apiserver logs when `auditlog_read_access: "true"`. Considering we only need this for a limited time to identify the users I think this is ok and I wouldn't make it more complicated than we need to.